apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -60,9 +64,23 @@ export default class VirtualizedDetailView extends Component {
         {(rowIndex === 0 && columnIndex === 0) && (
           <span style={styles.cell}>Lesson</span>
         )}
+        {(rowIndex === 0 && columnIndex >= 1) && (
+          <span style={styles.cell}>{columnIndex}</span>
+        )}
         {(rowIndex === 1 && columnIndex === 0) && (
           <span style={styles.cell}>Level Type</span>
         )}
+        {(rowIndex === 1 && columnIndex >= 1) && (
+          <span style={styles.cell}>
+            {scriptData.stages[columnIndex-1].levels.map((level, i) =>
+              <FontAwesome
+                className={level.icon ? level.icon: "fas fa-question"}
```

Comment:
This is hacky - we have some levels that have `null` for their icon value.  If that's the case, I use a question mark icon, which seems like it will be confusing in the long run.  What do we want to do if there isn't an icon associated with the level?

Reply:
I think that's fine for now, we'll address that once we have a prototype.